.github/workflows/ci.yml

```diff
@@ -70,6 +70,14 @@ jobs:
           file: coverage.out
           format: golang
 
+      - name: 'Upload junit-xml artifacts'
```

Comment:
nit: None of the other step names in this file wrap the strings with `'`

Reply:
Thanks! Removed.